solr/core/src/test-files/solr/collection1/conf/schema15.xml

```diff
@@ -567,7 +567,17 @@
   <field name="_root_" type="string" indexed="true" stored="true"/>
   <!-- required for NestedUpdateProcessor -->
   <field name="_nest_parent_" type="string" indexed="true" stored="true"/>
-  <field name="_nest_path_" type="string" indexed="true" stored="true"/>
+  <field name="_nest_path_" type="descendants_path" indexed="true" multiValued="false" docValues="true" stored="false" useDocValuesAsStored="true"/>
```

Comment:
I think useDocValuesAsStored=false.  This is internal and shouldn't be returned when someone does `fl=*`

Reply:
Sure thing, will fix ASAP